packages/bentley/imodel-react-hooks/package.json

```diff
@@ -53,6 +53,7 @@
     "@bentley/bentleyjs-core": "^2.0.0",
     "@bentley/geometry-core": "^2.0.0",
     "@bentley/imodeljs-common": "^2.0.0",
+    "@bentley/telemetry-client": "^2.0.0",
```

Comment:
I still don't understand why it fails all of a sudden without this. I guess maybe bad deps to begin with or maybe new dep added in core that wasn't caught :/. Oh well.

Reply:
yeah prob